python/runtime/cudaq/algorithms/py_draw.cpp

```diff
@@ -20,6 +20,9 @@ namespace cudaq {
 namespace details {
 std::tuple<std::string, MlirModule, OpaqueArguments *>
 getKernelLaunchParameters(py::object &kernel, py::args args) {
+  if (!py::hasattr(kernel, "arguments"))
+    throw std::runtime_error(
+        "unrecognized kernel - did you forget the @kernel attribute?");
```

Comment:
Might be better if it mentioned `arguments` as the specific attribute that is missing.